### ISSUE TITLE
Nerd fonts + Kitty 

### ISSUE THREAD

**Issue by Superredstone** [danth/stylix#665]
After updating my system (I'm on unstable) and doing the following change 

```
# Before 25.05
fonts.packages = with pkgs; [
  (nerdfonts.override { fonts = [ "FiraCode" "DroidSansMono" ]; })
];

# After 25.05
fonts.packages = with pkgs; [
  nerd-fonts.fira-code
  nerd-fonts.droid-sans-mono
]; 
```

I get the following error when trying to open Kitty (which I styled with Stylix): 

```
fast_data_types.FreeTypeError: Failed to load face from path: /etc/profiles/per-user/r3ddy/share/fonts/truetype/NerdFonts/JetBrainsMonoNerdFont-SemiBold.ttf with error: cannot open resource
```

I searched a little bit and found out that the Jetbrains font folder has been moved from` /etc/profiles/per-user/r3ddy/share/fonts/truetype/NerdFonts/JetBrainsMonoNerdFont-SemiBold.ttf` to `/etc/profiles/per-user/r3ddy/share/fonts/truetype/NerdFonts/JetBrainsMono/JetBrainsMonoNerdFont-SemiBold.ttf`

**Comment by mateusauler** [comment]
I had a similar error happen to me. I just ran `fc-cache -r`. Sadly it's imperative, so it has to be manually ran on every system; but only once.

**Comment by trueNAHO** [comment]
Closing this issue, as it seems to be a duplicate of https://github.com/danth/stylix/issues/650. Feel free to re-open, if this is not the case.
